src/components/tutor-popover.cjsx

```diff
@@ -0,0 +1,116 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+ArbitraryHtml = require './html'
+
+TutorPopover = React.createClass
+  displayName: 'TutorPopover'
+
+  getInitialState: ->
+    firstShow: true
+    placement: 'right'
+    show: false
+    scrollable: false
+    imageLoading: false
+
+  propTypes: ->
+    contentHtml: React.PropTypes.string.isRequired
+    popoverProps: React.PropTypes.object
+    contentProps: React.PropTypes.object
+    overlayProps: React.PropTypes.object
+    linkProps: React.PropTypes.object
+    windowImpl: React.PropTypes.object
+    maxHeightMultiplier: React.PropTypes.number
+
+  getDefaultProps: ->
+    maxHeightMultiplier: 0.75
+    windowImpl: window
+
+  componentDidMount: ->
+    @updateOverlayPositioning()
+
+  componentDidUpdate: ->
+    # Make sure the popover re-positions after the image loads
+    if @refs.popcontent? and @state.firstShow
+      content = @refs.popcontent.getDOMNode()
+      images = content.querySelectorAll('img')
+      for image in images
+        unless image.onload? or image.complete
+          @setState(imageLoading: true)
+          image.onload = @imageLoaded
```

Comment:
Should we care about the case when the link is to an element with multiple images?